compiler/src/dmd/dsymbolsem.d

```diff
@@ -4385,6 +4385,24 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             m.needmoduleinfo = 1;
             //printf("module1 %s needs moduleinfo\n", m.toChars());
         }
+
+        foreachUda(scd, sc, (Expression e) {
+            import dmd.attrib : isEnumAttribute;
+            if (!isEnumAttribute(e, Id.udaStandalone))
+                return 0;
+
+            if (auto sharedCtor = scd.isSharedStaticCtorDeclaration())
+            {
+                auto trust = sharedCtor.type.isTypeFunction().trust;
+                if (trust != TRUST.system && trust != TRUST.trusted)
+                    error(e.loc, "a module constructor using `@%s` must be `@system` or `@trusted`", Id.udaStandalone.toChars());
```

Comment:
why not `@standalone` directly ?

Reply:
There will be more of these in future, can it be made into a reusable function?

Reply:
to be frank, this is not a hot path. It's just that it's useless to add N kilobytes of code while the static data is already there.

Reply:
> can it be made into a reusable function

What part of the error would other attributes re-use?

Reply:
That's up to your judgement - "I just mimicked" hinted at some repetition. If there is no overlap then don't, but it would be really good to make these types of logic declarative. I'll write a sketch hang on.

Reply:
The 'repetition' is also using `@%s` and `Id.udaStandalone.toChars()` instead of `@standalone`, that's too small of a thing to extract to a function.